impala/hiveserver2.py

```diff
@@ -758,39 +759,55 @@ def threaded(func):
 
 def connect(host, port, timeout=None, use_ssl=False, ca_cert=None,
             user=None, password=None, kerberos_service_name='impala',
-            auth_mechanism=None, krb_host=None):
+            auth_mechanism=None, krb_host=None, use_http_transport=False,
+            http_path=''):
     log.debug('Connecting to HiveServer2 %s:%s with %s authentication '
               'mechanism', host, port, auth_mechanism)
-    sock = get_socket(host, port, use_ssl, ca_cert)
 
-    if krb_host:
-        kerberos_host = krb_host
+    if use_http_transport:
+        # TODO: Add server authentication with thrift 0.12
```

Comment:
Is there a thrift JIRA or something we can link to?

Reply:
Opened #362 and added it to the comment.